pytest_doctestplus/plugin.py

```diff
@@ -64,6 +63,14 @@ def pytest_addoption(parser):
                   default=1e-05)
 
 
+def get_optionflags(parent):
+    optionflags_str = parent.config.getini('doctest_optionflags')
+    flag_int = 0
+    for flag_str in optionflags_str:
+        flag_int |= doctest.OPTIONFLAGS_BY_NAME[flag_str]
```

Comment:
This does not appear to be equivalent to the flags that are supported by the built-in `doctest` plugin. I think what we might want to do instead is use `_pytest.doctest._get_flag_lookup()` to get the set of options used by the built-in `doctest` plugin, and then add the `FLOAT_CMP` option to that.

Reply:
This will use all the flags that have been registered with `doctest.register_optionflag`.

The `_get_flag_lookup` function could really be implemented as:
```python
def _get_flag_lookup():
    _get_allow_unicode_flag()
    _get_allow_bytes_flag()
    return doctest.OPTIONFLAGS_BY_NAME
```

The only flags that are not included by this patch are the `ALLOW_BYTES` and `ALLOW_UNICODE` flags. I'll add them in `outputchecker.py`.

Reply:
Actually neither of these options works with master either. It's not enough to list all the options from the pytest plugin if there isn't the code to implement them.

You can test with e.g.:

```python
def test_f():
    '''
    >>> 'asd'.encode('ascii') # doctest: +ALLOW_BYTES
    b'asd'
    '''
    pass
```

Now if you run that under Python 2.7 you'll see that it passes with `--doctest-modules` but fails with `--doctest-plus`.

I'll add them to `outputchecker.py` anyway so they can appear in doctests without crashing doctest-plus.

Reply:
I've pushed a new commit adding those two flags back

Reply:
Okay, this makes sense. I'm not sure whether we ought to support those or not, but if we do it should be added in a separate PR.